spdx-utils/build.gradle

```diff
@@ -128,16 +167,55 @@ task generateSpdxLicenseExceptionList {
 
         def enumValues = spdxLicenseExceptionIds.collect { "    ${licenseToEnumEntry(it.toString())}" }.sort().join(',\n') + ';'
         licenseExceptionEnumFile.append(enumValues)
-        licenseExceptionEnumFile.append("""
+        licenseExceptionEnumFile.append('''
 
                 companion object {
                     /**
                      * Return the enum value for the given [id], or null if it is no SPDX license exception id.
                      */
                     fun forId(id: String) = SpdxLicenseException.values().find { it.id == id }
                 }
+
+                /**
+                 * The full license exception text as a string.
+                 */
+                val text = javaClass.getResource("/exceptions/$id").readText()
```

Comment:
Consider using `by lazy { ... }` so that not all exception texts are read into memory when the class is loaded.